services/repository/repo_team.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
```

Comment:
I cannot understand why these functions should be moved from org service to repo service.

Reply:
Because we need to avoid the dependencies recycling, these functions have to be moved to the repository service.